src/ImageKit/ImageKit.php

```diff
@@ -419,13 +647,9 @@ public function getPurgeCacheStatus($requestId)
      *
      * @deprecated since 2.0.0, use <code>bulkDeleteFiles</code>
      */
-    public function bulkFileDeleteByIds($options)
+    public function bulkFileDeleteByIds($fileIds=null)
     {
-        if (!isset($options['fileIds'])) {
-            return Response::respond(true, ((object)ErrorMessages::$fileIdS_MISSING));
-        }
-
-        return $this->bulkDeleteFiles($options['fileIds']);
+        return $this->bulkDeleteFiles($fileIds);
```

Comment:
We can put size check on length of `fileIds`

Reply:
the mentioned method is deprecated, and the payload is transffered to the new method which has all the respective validations.